java/code/src/com/suse/manager/model/hub/HubManager.java

```diff
@@ -12,34 +12,59 @@
 package com.suse.manager.model.hub;
```

Comment:
Not sure if the "Manager" class is correctly placed in the "model" package.
I think we have "com.suse.manager.hub" or "com.suse.manager.iss" for Manager classes.
The model is for Database and Factory classes. Maybe for data models like json parser files.